src/conanfile.txt

```diff
@@ -0,0 +1,9 @@
+[requires]
```

Comment:
We rather need conanfile.py to create a ecfeed package and upload it to conancenter

Reply:
The package works locally. However, I'm not sure if it is a good idea to combine 'ecfeed' with 'picojson'.  We still need non-default libraries like 'libcurl', 'openssl' and one of the testing frameworks. If not provided, the program crushes. It can be done with Conan, but in this case, we can also add picojson as a dependency. Or maybe I'm missing something...

Reply:
OK I managed to declare 'libcurl' and 'openssl' as dependencies (it is not needed to include them in conanfile.txt). 'picojson' still looks weird though... There is one additional branch 'conancenter' used for the deployment.